modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -580,9 +580,9 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
 
         expect(diag.getSeverity == b.DiagnosticSeverity.ERROR)
         expect(diag.getRange.getStart.getLine == 0)
-        expect(diag.getRange.getStart.getCharacter == 3)
+        expect(diag.getRange.getStart.getCharacter == 16)
         expect(diag.getRange.getEnd.getLine == 0)
-        expect(diag.getRange.getEnd.getCharacter == 44)
+        expect(diag.getRange.getEnd.getCharacter == 16)
```

Comment:
The end character is a problem here (it should be 46 I guess)

Reply:
Yes, currently my lib returns position of the beginning of the element that you refer to. It needs some work to return correct span. Maybe we can leave it as it is and make it better in some follow-up PR?